app/models/binary/Publication.scala

```diff
@@ -18,7 +22,8 @@ case class Publication(_id: ObjectId,
                        created: Long = System.currentTimeMillis(),
                        isDeleted: Boolean = false)
 
-class PublicationService @Inject()()(implicit ec: ExecutionContext) {
+class CompactPublicationService @Inject()()(implicit ec: ExecutionContext) {
```

Comment:
I’d say this can be merged into one service, now that the dataset publicWrites no longer includes the publication :) cc @leowe